plugin/metrics/README.md

```diff
@@ -33,6 +34,12 @@ Extra labels used are:
   NS, SRV, DS, DNSKEY, RRSIG, NSEC, NSEC3, IXFR, AXFR and ANY) and "other" which lumps together all
   other types.
 * The `response_rcode_count_total` has an extra label `rcode` which holds the rcode of the response.
+* The `coredns_dns_programming_latency_seconds` metric has an extra `service_kind` label that
```

Comment:
this can be moved there as well, under a new section `## Metrics`

Reply:
Done